examples/options.txt

```diff
@@ -0,0 +1,15 @@
+generate command
```

Comment:
`needs review `

Reply:
Created a section named COMMANDS in README.md where all commands along with the options are described with examples.